tools/chef/roles/db-mysql55-client.json

```diff
@@ -3,6 +3,18 @@
   "chef_type": "role",
   "json_class": "Chef::Role",
   "description": "MySQL 5.5 database client",
+  "default_attributes": {
+    "mysql-libs": {
+      "replace_package_targets": [
+        "mysql55-libs",
+        "Percona-Server-shared-51",
+        "mysqlclient16"
+      ]
+    },
+    "mysql-client": {
+      "replace_package_target": "mysql55"
+    }
+  },
```

Comment:
Fair point, we can move the "enabled" flag into each role, defaulting to false.
I'll also add a guard into that recipe.